2022/AzureToronto/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "ValidMaintainers": {
+      "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+      "GitHubHandles": [
+        "eskandari"
+      ]
+    },
+    "CommunityInfo": {
+      "Name": "Azure Toronto",
+      "Country": "Canada",
+      "Url": "https://www.meetup.com/metrotorontoug/events/283781596/",
+      "SessionizeApiId": "67mp1zuj",
```

Comment:
It looks to me like you have not configured sessionize correctly. Please review the instruction here under point 5! https://github.com/globalazure/communities/blob/main/README.md

Reply:
Thanks @noopman  fro all the efforts.
index.md added
_template.png removed
sessionize api id removed